app/components/course-page/step-list.hbs

```diff
@@ -1,5 +1,39 @@
 <div class="flex flex-col gap-y-1" ...attributes>
-  {{#each @stepList.stepGroups as |stepGroup|}}
+  {{#if this.baseStagesStepGroup.title}}
+    <div class="px-3 py-2">
+      <h3 class="text-xs uppercase font-semibold text-gray-400 border-b">{{this.baseStagesStepGroup.title}}</h3>
+    </div>
+  {{/if}}
+
+  {{#each this.baseStagesStepGroup.visibleSteps as |step|}}
+    <CoursePage::StepListItem @step={{step}} @isCurrent={{eq this.currentStep step}} @isActive={{eq this.activeStep step}} />
+  {{/each}}
+
+  {{#if (gt @course.extensions.length 0)}}
+    <div class="flex justify-center mt-3 px-3">
```

Comment:
```suggestion
    <div class="flex justify-center my-3 px-3">
```

Looks better with even space on both ends: 

<img width="302" alt="Screenshot 2024-03-05 at 18 14 56" src="https://github.com/codecrafters-io/frontend/assets/3893573/79dcf5f8-129d-40d8-b3ce-e4e97e54c5c9">

This wasn't needed earlier since the button was at the bottom of the list.